common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -492,6 +596,23 @@ def test_user_cannot_remove_default_image(self):
         self.assert_default_image_has_public_access(profile_page)
         self.assertFalse(profile_page.remove_link_present)
 
+    def test_eventing_after_multiple_uploads(self):
+        """
+        Scenario: An event is fired when a user with a profile image uploads another image
+
+        Given that I am on my profile page with public access
+        And I upload a new image via file uploader
+        When I upload another image via the file uploader
+        Then two upload events have been emitted
+        """
+        username, user_id = self.log_in_as_unique_user()
+        profile_page = self.visit_profile_page(username, privacy=self.PRIVACY_PUBLIC)
+        self.assert_default_image_has_public_access(profile_page)
+        profile_page.upload_file(filename='image.jpg')
+        self.assertTrue(profile_page.image_upload_success)
+        profile_page.upload_file(filename='image.jpg', wait_for_upload_button=False)
```

Comment:
@symbolist and/or @muzaffaryousaf We had a bug in the profile image eventing code that caused uploads to break if you uploaded an image after you already had one. I wanted to add a test case for this. The bok choy test did not previously have a test for changing an existing profile image.

I found that I couldn't wait for the upload button visibility on the second upload, but not understanding the UI code that well, it is unclear to me why this is the case. I disabled the check on the second upload (wait_for_upload_button=False), but just wanted to run it by you guys.